Dynamo/Dynamo.SwiftLang/SLClass.cs

```diff
@@ -105,6 +110,8 @@ static SLIdentifier IdentifierForNamedType (NamedType nt)
 					decl.Add (new SimpleElememt ("static ", true));
 				if (IsSealed)
 					decl.Add (new SimpleElememt ("sealed ", true));
+				if (IsFinal)
+					decl.Add (new SimpleElememt ("final ", true));
```

Comment:
`SimpleElememt` -> `SimpleElement`
not clear why I never spotted it before... and better fixed in a different PR

Reply:
Oh, that's hilarious. I opened an issue for it [here](https://github.com/xamarin/binding-tools-for-swift/issues/369) for you to fix.